### PR TITLE
Server Entity Attachment: Update checks

### PR DESCRIPTION
**Pull request by Antho1426RR**
Features:
- Remove server check for attaching/detaching items at will (temporary).
- Add check for mobility of pickable objects (which as to be set as `Movable`).

### CODE REVIEW

File: Source/RapyutaSimulationPlugins/Private/Tools/SimulationState.cpp [hunk 1]
```diff
@@ -255,39 +255,52 @@ void ASimulationState::ServerAttach(const FROSAttachReq& InRequest)
         return;
     }
 
-    if (ServerCheckAttachRequest(InRequest))
+    // TODO: Add proper server check
+    //if (ServerCheckAttachRequest(InRequest))
+    if (true)
```
Comment (yuokamoto) [inline Source/RapyutaSimulationPlugins/Private/Tools/SimulationState.cpp]: just remove if(true) but keep comment

**Review by yuokamoto — APPROVED** [review]

**Review by Tadinu — APPROVED** [review]

**Comment by Tadinu** [comment]
@Antho1426RR seems like you cherry-pick commits already merged to devel & some from #144?

**Comment by yuokamoto** [comment]
please rebase from devel and remove cherry-pick of separate_odom

**Comment by Antho1426RR** [comment]
Equivalent PR is at "[Move pallets scenario 2 #145](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/pull/145)".

**Pull request closed without merging**